TUTORIAL.md

```diff
@@ -213,17 +213,17 @@ let timerTick (dispatch: Dispatch<Msg>) =
   timer.Start()
 ```
 
-The final alias, `Cmd<'msg>`, is just a list of `Sub<'msg>`, i.e. a list of `Dispatch<'msg> -> unit` functions. In other words, the `update` function can return a list of `Dispatch<'msg> -> unit` functions that the MVU update loop will execute. These functions, as you saw above, can dispatch any message at any time. Therefore, if you need to do impure stuff such as calling a web API, you simply create a function accepting `dispatch`, perform the call there (likely using `async`), and use the `dispatch` argument to dispatch a message when you receive a response.
+The final alias, `Cmd<'msg>`, is just a list of `Sub<'msg>`, i.e. a list of `Dispatch<'msg> -> unit` functions. In other words, the `update` function can return a list of `Dispatch<'msg> -> unit` functions that the MVU will execute by providing a dispatch function. These commands functions, as you saw above, can then dispatch any message at any time. Therefore, if you need to do impure stuff such as calling a web API, you simply create a function accepting `dispatch`, perform the work within it asynchronously, and then use the `dispatch` argument it will have been provided to insert the response into the MVU event loop.
```

Comment:
> that the MVU will execute

"That the MVU framework will execute"

> These commands functions

Typo (should be singular "command"), but anyway, I think that's confusing. **A command** is **a list of functions**. Here you are essentially saying that **a command** is **a list of command functions**. I suggest removing "commands".

> perform the work within it asynchronously

Let's make it more general and precise: "perform the impure work within it"

> and then use the `dispatch` argument it will have been provided to insert the response into the MVU event loop

Suggested re-phrasing: "and then use the `dispatch` argument (provided by the MVU framework) to dispatch further messages (e.g. representing the result of the action) into the MVU event loop"